p4proto/p4src/linux_networking/README_LINUX_NETWORKING.md

```diff
@@ -117,14 +117,25 @@ Note:
     ip addr add 99.0.0.2/24 dev eth0
     ip link set dev eth0 up
 ```
-#### 7. Bring up the TAP interfaces and assign IP to TAP interfaces
-```
+#### 7. Bring up the TAP and or TEP interfaces
+  - 7.1 Use one of the TAP ports as tunnel termination and assign IP address to the TAP port.
+  ```
     ip link set dev TAP0 up
     ip addr add 40.1.1.1/24 dev TAP1
     ip link set dev TAP1 up
     ip link set dev TAP2 up
     ip link set dev TAP3 up
-```
+  ```
+  - 7.2 Create a TEP port and use this TEP port for tunnel termination
```

Comment:
Better to use the dummy device terminology instead of "TEP".
Also add additional detail on approach which uses dummy port.